components/admin_console/custom_plugin_settings/index.js

```diff
@@ -27,13 +31,35 @@ function makeGetPluginSchema() {
             let settings = [];
             if (plugin.settings_schema && plugin.settings_schema.settings) {
                 settings = plugin.settings_schema.settings.map((setting) => {
+                    const key = setting.key.toLowerCase();
+                    let component = null;
+                    let bannerType = '';
+                    let type = setting.type;
+                    let displayName = setting.display_name;
+                    let isDisabled = it.stateIsFalse(pluginEnabledConfigKey);
+
+                    if (customComponents[key]) {
+                        component = customComponents[key].component;
+                        displayName = customComponents[key].title;
+                        type = Constants.SettingsTypes.TYPE_CUSTOM;
+                    } else if (!customComponents[key] && setting.type === Constants.SettingsTypes.TYPE_CUSTOM) {
```

Comment:
I think this `!customComponents[key]` check here is redundant, the `} else if` includes already that meaning, right?

Reply:
Correct. Removed